src/utils/survey.ts

```diff
@@ -0,0 +1,29 @@
+import { OwnerInformation, Survey, Building } from "generated/client";
+import { SurveyWithInfo } from "types";
+
+/**
+ * Utility class for survey
+ */
+export default class SurveyUtils {
+
+  /**
+   * Parse survey with information
+   */
+  public static surveyWithInfoParser(survey: Survey, building?: Building, ownerInformation?: OwnerInformation) {
```

Comment:
- No need to explicitly define "public" since in Typescript, class members are public by default
- Should be named by what it does, not what it is, e.g. "parseToSurveyWithInfo" or something similar.